### ISSUE TITLE
[Help] API Limit Reached

### ISSUE THREAD

**Issue by ItsJohnny30** [JaxkDev/DiscordBot#86]
```
[2023-02-16T17:50:24.666587+00:00] DiscordThread.DEBUG: BUCKET getgateway/bot queued REQ GET gateway/bot [] []
[2023-02-16T17:50:24.837599+00:00] DiscordThread.CRITICAL: ErrorException: "Undefined property: stdClass::$global" (EXCEPTION) in "phar:///home/container/plugins/DiscordBot_v2_1_6.phar/vendor/discord-php/http/src/Discord/Http.php" at line 319 [] []
[2023-02-16T17:50:24.839961+00:00] DiscordThread.CRITICAL: #0 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/discord-php/http/src/Discord/Http(319): JaxkDev\DiscordBot\Bot\Client->sysErrorHandler(integer 2, string[37] Undefined property: stdClass::$global, string[98] phar:///home/container/plugins/DiscordBot_v2_1_6.phar/vendor/discord-php/http/sr, integer 319) [] []
[2023-02-16T17:50:24.840037+00:00] DiscordThread.CRITICAL: #1 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/FulfilledPromise(42): Discord\Http\Http->Discord\Http\{closure}(object RingCentral\Psr7\Response#2862) [] []
[2023-02-16T17:50:24.840064+00:00] DiscordThread.CRITICAL: #2 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(66): React\Promise\FulfilledPromise->done(object Closure#3383, object Closure#2871) [] []
[2023-02-16T17:50:24.840084+00:00] DiscordThread.CRITICAL: #3 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(168): React\Promise\Promise::React\Promise\{closure}(object React\Promise\FulfilledPromise#3230) [] []
[2023-02-16T17:50:24.840101+00:00] DiscordThread.CRITICAL: #4 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(231): React\Promise\Promise->settle(object React\Promise\FulfilledPromise#3230) [] []
[2023-02-16T17:50:24.840118+00:00] DiscordThread.CRITICAL: #5 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Deferred(36): React\Promise\Promise::React\Promise\{closure}(object RingCentral\Psr7\Response#2862) [] []
[2023-02-16T17:50:24.840134+00:00] DiscordThread.CRITICAL: #6 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/http/src/Io/Transaction(89): React\Promise\Deferred->resolve(object RingCentral\Psr7\Response#2862) [] []
[2023-02-16T17:50:24.840150+00:00] DiscordThread.CRITICAL: #7 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/FulfilledPromise(28): React\Http\Io\Transaction->React\Http\Io\{closure}(object RingCentral\Psr7\Response#2862) [] []
[2023-02-16T17:50:24.840166+00:00] DiscordThread.CRITICAL: #8 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(134): React\Promise\FulfilledPromise->then(object Closure#2793, object Closure#3363) [] []
[2023-02-16T17:50:24.840182+00:00] DiscordThread.CRITICAL: #9 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(168): React\Promise\Promise::React\Promise\{closure}(object React\Promise\FulfilledPromise#2896) [] []
[2023-02-16T17:50:24.840197+00:00] DiscordThread.CRITICAL: #10 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(231): React\Promise\Promise->settle(object React\Promise\FulfilledPromise#2896) [] []
[2023-02-16T17:50:24.840219+00:00] DiscordThread.CRITICAL: #11 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/FulfilledPromise(42): React\Promise\Promise::React\Promise\{closure}(object RingCentral\Psr7\Response#2862) [] []
[2023-02-16T17:50:24.840235+00:00] DiscordThread.CRITICAL: #12 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(135): React\Promise\FulfilledPromise->done(object Closure#3418, object Closure#2939, object Closure#3158) [] []
[2023-02-16T17:50:24.840250+00:00] DiscordThread.CRITICAL: #13 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(168): React\Promise\Promise::React\Promise\{closure}(object React\Promise\FulfilledPromise#3159) [] []
[2023-02-16T17:50:24.840266+00:00] DiscordThread.CRITICAL: #14 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(231): React\Promise\Promise->settle(object React\Promise\FulfilledPromise#3159) [] []
[2023-02-16T17:50:24.840281+00:00] DiscordThread.CRITICAL: #15 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/FulfilledPromise(42): React\Promise\Promise::React\Promise\{closure}(object RingCentral\Psr7\Response#2862) [] []
[2023-02-16T17:50:24.840296+00:00] DiscordThread.CRITICAL: #16 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(66): React\Promise\FulfilledPromise->done(object Closure#3252, object Closure#2966) [] []
[2023-02-16T17:50:24.840318+00:00] DiscordThread.CRITICAL: #17 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(168): React\Promise\Promise::React\Promise\{closure}(object React\Promise\FulfilledPromise#2901) [] []
[2023-02-16T17:50:24.840334+00:00] DiscordThread.CRITICAL: #18 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(231): React\Promise\Promise->settle(object React\Promise\FulfilledPromise#2901) [] []
[2023-02-16T17:50:24.840350+00:00] DiscordThread.CRITICAL: #19 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/FulfilledPromise(42): React\Promise\Promise::React\Promise\{closure}(object RingCentral\Psr7\Response#2862) [] []
[2023-02-16T17:50:24.840365+00:00] DiscordThread.CRITICAL: #20 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(135): React\Promise\FulfilledPromise->done(object Closure#3360, object Closure#2863, object Closure#3391) [] []
[2023-02-16T17:50:24.840380+00:00] DiscordThread.CRITICAL: #21 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(168): React\Promise\Promise::React\Promise\{closure}(object React\Promise\FulfilledPromise#329) [] []
[2023-02-16T17:50:24.840395+00:00] DiscordThread.CRITICAL: #22 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(231): React\Promise\Promise->settle(object React\Promise\FulfilledPromise#329) [] []
[2023-02-16T17:50:24.840415+00:00] DiscordThread.CRITICAL: #23 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/FulfilledPromise(42): React\Promise\Promise::React\Promise\{closure}(string[238] {"code": 0, "message": "You are being blocked from accessing our API temporarily) [] []
[2023-02-16T17:50:24.840432+00:00] DiscordThread.CRITICAL: #24 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(135): React\Promise\FulfilledPromise->done(object Closure#3168, object Closure#2935, object Closure#2996) [] []
[2023-02-16T17:50:24.840466+00:00] DiscordThread.CRITICAL: #25 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(168): React\Promise\Promise::React\Promise\{closure}(object React\Promise\FulfilledPromise#2846) [] []
[2023-02-16T17:50:24.840483+00:00] DiscordThread.CRITICAL: #26 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise/src/Promise(231): React\Promise\Promise->settle(object React\Promise\FulfilledPromise#2846) [] []
[2023-02-16T17:50:24.840498+00:00] DiscordThread.CRITICAL: #27 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/promise-stream/src/functions(79): React\Promise\Promise::React\Promise\{closure}(string[238] {"code": 0, "message": "You are being blocked from accessing our API temporarily) [] []
[2023-02-16T17:50:24.840514+00:00] DiscordThread.CRITICAL: #28 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/evenement/evenement/src/Evenement/EventEmitterTrait(123): React\Promise\Stream\{closure}() [] []
[2023-02-16T17:50:24.840576+00:00] DiscordThread.CRITICAL: #29 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/http/src/Io/ReadableBodyStream(50): Evenement\EventEmitter->emit(string[5] close) [] []
[2023-02-16T17:50:24.840594+00:00] DiscordThread.CRITICAL: #30 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/http/src/Io/ReadableBodyStream(151): React\Http\Io\ReadableBodyStream->close() [] []
[2023-02-16T17:50:24.840609+00:00] DiscordThread.CRITICAL: #31 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/evenement/evenement/src/Evenement/EventEmitterTrait(123): React\Http\Io\ReadableBodyStream->handleEnd() [] []
[2023-02-16T17:50:24.840625+00:00] DiscordThread.CRITICAL: #32 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/http/src/Io/ChunkedDecoder(150): Evenement\EventEmitter->emit(string[3] end) [] []
[2023-02-16T17:50:24.840640+00:00] DiscordThread.CRITICAL: #33 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/evenement/evenement/src/Evenement/EventEmitterTrait(123): React\Http\Io\ChunkedDecoder->handleData(string[249] ee..{"code": 0, "message": "You are being blocked from accessing our API tempora) [] []
[2023-02-16T17:50:24.840682+00:00] DiscordThread.CRITICAL: #34 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/http/src/Client/Request(163): Evenement\EventEmitter->emit(string[4] data, array[1]) [] []
[2023-02-16T17:50:24.840705+00:00] DiscordThread.CRITICAL: #35 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/evenement/evenement/src/Evenement/EventEmitterTrait(123): React\Http\Client\Request->handleData(string[963] HTTP/1.1 429 Too Many Requests..Date: Thu, 16 Feb 2023 17:50:24 GMT..Content-Typ) [] []
[2023-02-16T17:50:24.840722+00:00] DiscordThread.CRITICAL: #36 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/stream/src/Util(71): Evenement\EventEmitter->emit(string[4] data, array[1]) [] []
[2023-02-16T17:50:24.840737+00:00] DiscordThread.CRITICAL: #37 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/evenement/evenement/src/Evenement/EventEmitterTrait(123): React\Stream\Util::React\Stream\{closure}(string[963] HTTP/1.1 429 Too Many Requests..Date: Thu, 16 Feb 2023 17:50:24 GMT..Content-Typ) [] []
[2023-02-16T17:50:24.840753+00:00] DiscordThread.CRITICAL: #38 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/stream/src/DuplexResourceStream(196): Evenement\EventEmitter->emit(string[4] data, array[1]) [] []
[2023-02-16T17:50:24.840769+00:00] DiscordThread.CRITICAL: #39 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/event-loop/src/StreamSelectLoop(246): React\Stream\DuplexResourceStream->handleData(resource (closed) Resource id #959) [] []
[2023-02-16T17:50:24.840784+00:00] DiscordThread.CRITICAL: #40 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/react/event-loop/src/StreamSelectLoop(213): React\EventLoop\StreamSelectLoop->waitForStreamActivity(integer 49999) [] []
[2023-02-16T17:50:24.840814+00:00] DiscordThread.CRITICAL: #41 /home/container/plugins/DiscordBot_v2_1_6.phar/vendor/team-reflex/discord-php/src/Discord/Discord(1368): React\EventLoop\StreamSelectLoop->run() [] []
[2023-02-16T17:50:24.840832+00:00] DiscordThread.CRITICAL: #42 /home/container/plugins/DiscordBot_v2_1_6.phar/src/Bot/Client(111): Discord\Discord->run() [] []
[2023-02-16T17:50:24.841276+00:00] DiscordThread.CRITICAL: #43 /home/container/plugins/DiscordBot_v2_1_6.phar/src/Communication/BotThread(51): JaxkDev\DiscordBot\Bot\Client->__construct(object JaxkDev\DiscordBot\Communication\BotThread#1, array[4]) [] []
[2023-02-16T17:50:24.841303+00:00] DiscordThread.CRITICAL: #44 (): JaxkDev\DiscordBot\Communication\BotThread->run() [] []
[2023-02-16T17:50:24.841540+00:00] DiscordThread.INFO: discord closed [] []
[2023-02-16T17:50:24.841711+00:00] DiscordThread.NOTICE: Discord thread closed. [] []
```

WTF THE PLUGIN??

**Comment by JaxkDev** [comment]
This is not a plugin issue, you’re spamming the API with requests too fast and you’ve been temporarily blocked by Discord.

**Comment by ItsJohnny30** [comment]
How to fix this? Help

**Comment by ItsJohnny30** [comment]
What? I'm didn't spamming the API with request... HELP ME bro!

**Comment by JaxkDev** [comment]
Whats your code, using the DiscordBot API

**Comment by ItsJohnny30** [comment]
> Whats your code, using the DiscordBot API

Do you have discord? Can I dm you?

**Comment by JaxkDev** [comment]
JaxkDev#2698

**Comment by ItsJohnny30** [comment]
ok, thx bro.
